dev/docker-clients/client.nomad

```diff
@@ -0,0 +1,52 @@
+job "client" {
+  datacenters = ["dc1"]
+
+  group "client" {
+    count = 6
+
+    task "agent" {
+      driver = "docker"
+
+      config {
+        image = "djenriquez/nomad:v0.6.0"
+
+        # command = "nomad"
+        args         = ["agent"]
+        network_mode = "host"
+        volumes      = ["local/config:/etc/nomad", "/var/run/docker.sock:/var/run/docker.sock", "/tmp:/tmp"]
```

Comment:
So that you can access the clients directories without using the fs command. Just easier debugging.